tiny_dnn/io/caffe/layer_factory_impl.h

```diff
@@ -660,7 +674,7 @@ inline std::shared_ptr<layer> create_convlayer(
   }
 
   auto conv = std::make_shared<conv_layer>(
-    in_width, in_height, window_size, in_channels, out_channels, table,
+    in_width, in_height, window_size_h, window_size_w, in_channels, out_channels, table,
```

Comment:
when i use the caffe_convert tools to convert my caffe model to tiny-dnn,  an error occured, becasue a convolutional layer which shape is 2 x 3 is not supported, with the Error message is :"unsupported kernel shape" in line 97. So i avoid the check in "get_kernel_size_2d" and directly to get the kernel_h and kernal_w  from "conv_param".  finaly, the conv_layer with 2*3 size is created successful.